StardewArchipelago/Locations/Patcher/ModLocationPatcher.cs

```diff
@@ -92,26 +90,8 @@ private void AddModSkillInjections()
                 postfix: new HarmonyMethod(typeof(RecipeLevelUpInjections), nameof(RecipeLevelUpInjections.SkillLevelUpMenuConstructor_SendModdedSkillRecipeChecks_Postfix))
             );
 
-            InjectSocializingExperienceMultiplier();
-            InjectArchaeologyExperienceMultiplier();
-        }
-
-        private void InjectSpaceCoreSkillsPage()
-        {
-            if (!_modsManager.ModIsInstalledAndLoaded(_modHelper, "SpaceCore"))
-            {
-                return;
-            }
-
-            var spaceCoreSkillsPageType = AccessTools.TypeByName("SpaceCore.Interface.NewSkillsPage");
-            var desiredNewSkillsPageCtorParameters = new[] { typeof(int), typeof(int), typeof(int), typeof(int) };
-            _harmony.Patch(
-                original: AccessTools.Constructor(spaceCoreSkillsPageType, desiredNewSkillsPageCtorParameters),
-                prefix: new HarmonyMethod(typeof(NewSkillsPageInjections),
-                    nameof(NewSkillsPageInjections.NewSkillsPageCtor_BearKnowledgeEvent_Prefix)),
-                postfix: new HarmonyMethod(typeof(NewSkillsPageInjections),
-                    nameof(NewSkillsPageInjections.NewSkillsPageCtor_BearKnowledgeEvent_Postfix))
-            );
+            //InjectSocializingExperienceMultiplier();

```

Comment:
Are we getting rid of this?

Reply:
Yeah.  If you recall, it was a method we had to patch because SpaceCore's inventory UI was different from Vanilla, and there was a visual discrepency on whether or not the paw showed up.  There's a unified UI now that Spacecore uses anyway.

Reply:
I meant the Socializing and Archaeology xp multipliers

Reply:
Oh.  Probably not?  It was just erroring and wasn't sure how to fix it yet.

Reply:
You can ignore them completely now (after a rebase), as I have fixed these two